target/linux/ramips/image/mt7620.mk

```diff
@@ -430,6 +430,26 @@ define Device/dch-m225
 endef
 TARGET_DEVICES += dch-m225
 
+define Device/kn_rc
+  DTS := kn_rc
+  IMAGE_SIZE := $(ralink_default_fw_size_8M)
```

Comment:
Can be dropped, the default image size when not specified otherwise is 8M: https://github.com/13hakta/source/blob/57889558bfba7247ddf14be7d48c3067e7bb1ef6/target/linux/ramips/image/Makefile#L39